src/lang/en.json

```diff
@@ -53,6 +53,9 @@
   "error": "Error",
   "error_message": "Whoops! Something went wrong.",
   "error_four_oh_four_message": "The page you are looking for cannot be found.",
+
+  "unknown": "Unknown",
```

Comment:
these strings should probably be namespaced under `general_` and moved into alphabetical order (by namespace).